build.gradle

```diff
@@ -0,0 +1,21 @@
+apply plugin: 'java'
+apply plugin: 'war'
+
+repositories {
+    jcenter()
+    mavenCentral()
+    mavenLocal()
+}
+
+dependencies {
+    compile fileTree(dir: 'lib', include: '*.jar')
+    compile 'javax.servlet:javax.servlet-api:3.0.1'
+    compile 'postgresql:postgresql:9.1-901.jdbc4'
+
+    testCompile 'simple-jndi:simple-jndi:0.11.4.1'
+    testCompile 'junit:junit:4.12'
+}
+
+sourceSets.main.java.srcDirs = ["src"]
+// the tests fail because they have environemnt specific parameters
+// sourceSets.test.java.srcDirs = ["test"]
```

Comment:
Yes, I have seen this possibility in the Gradle documentation. However, the TAP library needs the two others. So if I do this "source split" I would have to duplicate the adql and uws sources or to import them in some way.

I still do not know what to do. Maybe the simplest would be to have a general Gradle build file for the whole project, and then, keep my ANT scripts to check, build and publish (here, generate the JAR files) each library. Well....I will continue to think about that. For sure, it would have been simpler if initially the three libraries were split in different repositories and that TAPLib merely imported ADQLLib and UWSLib as sub-modules using Git. It still possible, but like the multi-project build proposed by Gradle, it means that users of this Github repository would have to perform some actions on their side.